src/components/modals/PaymentDetailsDialog.vue

```diff
@@ -497,11 +505,29 @@ export default Vue.extend({
       if (this.isAcceptedAllAgreements) {
         await this.processPayment();
       } else {
-        notifier.show({
-          message: this.$t('billing.paymentDetails.didNotAccept') as string,
-          style: 'error',
-          time: 5000,
-        });
+        if(this.isRecurrent){
+           if(!this.isAcceptedRecurrentPaymentAgreement){
+              notifier.show({
+                message: this.$t('billing.paymentDetails.didNotAcceptRecurrentPaymentAgreement') as string,
+                style: 'error',
+                time: 5000,
+              });
+           }
+           if(!this.isAcceptedChargingEveryMonth){
+              notifier.show({
+                message: this.$t('billing.paymentDetails.didNotAcceptChargingEveryMonth') as string,
+                style: 'error',
+                time: 5000,
+              });             
+           }
+        }
+        else{
```

Comment:
pls lint the code via ESLint

Reply:
@neSpecc,  there is no lint in this file.
![image](https://user-images.githubusercontent.com/23169768/130458367-a2dc7c7b-b21d-446b-95b2-bf902aa04ec9.png)